core/test/com/mygdx/config/GdxBaseTest.java

```diff
@@ -0,0 +1,25 @@
+package com.mygdx.config;
+
+import com.badlogic.gdx.Gdx;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+class GdxBaseTest {
+
+    @Test
```

Comment:
Nie jestem przekonany do tej konwencji nazywania testów, byłbym bardziej za shouldDoSomething, albo coś tego typu UnitOfWork_StateUnderTest_ExpectedBehavior

Reply:
a mi się podoba

Reply:
zrobiłem na to issue do obgadania